docs/driver.md

```diff
@@ -12,14 +12,20 @@ The command used to launch the IDE. By default, the driver uses the `idea` comma
 By default, the driver waits `30 seconds` for the IDE to connect the probe. After exceeding this time, the whole test fails.  
 #### Automatic checks
 ```
-driver.checks {
-    errors = false
+driver.check {
+     errors {
+       enabled = false
+       exclude-messages = []
+       include-messages = [".*"]
+     }
     freezes = false
```

Comment:
While you're at this, to prevent introducing yet another breaking release in future... pls change so that `freezes` is an object with `enabled` field as well. The other fields may (or may not) be added in future (but without breaking anything)